frontend/web/main.js

```diff
@@ -91,8 +91,8 @@ ReactDOM.render(<ToastMessages/>, document.getElementById('toast'));
 if (E2E) {
     document.body.classList.add('disable-transitions');
 }
-
-if (!E2E && Project.crispChat) {
+const isWidget = document.location.href.includes("/datadog")
```

Comment:
I noticed earlier we had some logic based on /widget, but here we're basing it on /datadog. Is there a reason for this?